gtp2ogs.js

```diff
@@ -520,14 +523,17 @@ class Connection {
             if (gamedata.phase == "play" && gamedata.player_to_move == this.bot_id) {
                 this.processMove(gamedata);
             }
-
-
-            if (this.connected_game_timeouts[gamedata.game_id]) {
-                clearTimeout(this.connected_game_timeouts[gamedata.game_id])
+            if (gamedata.phase == "finished") {
+                this.disconnectFromGame(gamedata.id);
```

Comment:
I think we need to potentially do a `clearTimeout` here to clear the timer that gets created in the else block